domains/jonee2.json

```diff
@@ -7,6 +7,6 @@
         "twitter": "Jonee1155"
     },
     "record": {
-        "CNAME": "jhonneg.github.io"
+        "CNAME": "jonee2.github.io"
```

Comment:
I cannot reach your website (404 page)

Reply:
changed link and deleted a file, please try again.